src/style/reset.less

```diff
@@ -47,9 +47,14 @@ footer, header, hgroup, menu, nav, section {
   display: block;
 }
 
-body {
+body { //this will get overridden by bootstrap?
```

Comment:
The order styles get included is (take a look at index.js):
- reset.less
- bootstrap
- style.less

So anything you want to keep defined on a global level goes to `style.less`.  `reset.less` merely resets the default browser styles.